Formula/i386-elf-binutils.rb

```diff
@@ -0,0 +1,24 @@
+class I386ElfBinutils < Formula
+  desc "FSF Binutils for i386-elf cross development"
+  homepage "https://www.gnu.org/software/binutils/"
+  url "https://ftp.gnu.org/gnu/binutils/binutils-2.31.tar.gz"
+  sha256 "5a9de9199f22ca7f35eac378f93c45ead636994fc59f3ac08f6b3569f73fcf6f"
+  depends_on "gettext" => :build
```

Comment:
Blank line before the first `depends_on`.
I don't think `gettext` and `xz` are actual dependencies.